modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -10,28 +10,23 @@ To connect the sample application to the database service, you must create a `Se
 . Create a `ServiceBinding` CR to project the binding data:
 +
 ----
```

Comment:
I noticed that the openshift-docs use `[source,terminal]` consistently above command examples. How does using that line affect the appearance of the command in the published documentation?
```suggestion
[source,terminal]
----
```

Reply:
Would it help to use it consistently with the commands and output examples in this PR?

Reply:
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output